app/views/inflation_proof.scala.html

```diff
@@ -15,11 +15,14 @@
  *@
 
 @import config.ApplicationConfig
-@(inflationProofForm: Form[models.InflationProof])(implicit request: Request[_], context: config.GmpContext, messages: Messages,applicationConfig: ApplicationConfig)
-
 @import uk.gov.hmrc.play.views.html._
 @import uk.gov.hmrc.play.views.html.helpers._
 @import views.html.helpers._
```

Comment:
I think this import can go now

Reply:
There's quite a few instances of this. You might be able to do a global find and replace

Reply:
Removed the play views imports in all views